products/flux2.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Flux
+category: server-app
+tags: cncf
+iconSlug: flux2
+permalink: /flux
+alternate_urls:
+-   /flux2
+-  /fluxcd
+versionCommand: flux version
+releasePolicyLink: https://fluxcd.io/flux/releases/#supported-releases
+changelogTemplate: https://github.com/fluxcd/flux2/releases/tag/v__LATEST__
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+-   git: https://github.com/fluxcd/flux2.git
+
+# eol(X) = releaseDate(X+3)
+releases:
```

Comment:
Even if it's now deprecated / does not share the same release / support policy, it would be great to have information about Flux 1.